plugins/amazonq/mynah-ui/src/mynah-ui/ui/connector.ts

```diff
@@ -594,35 +603,4 @@ export class Connector {
                 break
         }
     }
-
-    onFormButtonClick = (
-        tabId: string,
-        messageId: string,
-        action: {
-            id: string
-            text?: string
-            formItemValues?: Record<string, string>
-        }
-    ) => {
-        if (isFormButtonCodeTransform(action.id)) {
-            this.codeTransformChatConnector.onFormButtonClick(tabId, action)
-        } else if (isFormButtonCodeScan(action.id)) {
-            this.codeScanChatConnector.onFormButtonClick(tabId, action)
-        } else if (isFormButtonCodeTest(action.id)) {
-            this.codeTestChatConnector.onFormButtonClick(tabId, messageId, action)
-        } else if (action.id === 'doc') {
-            this.docChatConnector.onFormButtonClick(tabId, action)
-        }
```

Comment:
it's moved to here https://github.com/aws/aws-toolkit-jetbrains/pull/5286/files#diff-b5589403179a83b21f2b80bb57e8ab810c218b57327eaf711f494c033a790ba9L545-L561